cli/monochrome_cli/core/search.py

```
@@ -0,0 +1,255 @@
+                            disc_number=int(item.get("volumeNumber", 1)),
+                            year=year,
+                            release_date=release_date,
+                            isrc=item.get("isrc"),
+                            explicit=bool(item.get("explicit", False)),
+                            cover_url=cover_url,
+                            source="tidal",
+                            source_id=str(item.get("id"))
+                        ))
+                    return tracks
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)